catalyst_voices/packages/catalyst_voices_models/lib/src/user/user.dart

```diff
@@ -1,17 +1,26 @@
+import 'package:catalyst_voices_models/src/user/profile.dart';
 import 'package:equatable/equatable.dart';
 
-/// Defines the profile of the app user.
+/// Defines user or the app.
 final class User extends Equatable {
-  final String name;
+  final List<Profile> profiles;
```

Comment:
not entirely sure why a user would have multiple profiles... they can belong to multiple teams (in the future, etc) but what are multiple profiles?

Reply:
Okay I see your point, though it's not defined if users will create separate keychains per brand or reuse the same one.